store/todos.js

```diff
@@ -124,4 +148,33 @@ export const mutations = {
     state.perPage = PER_PAGE;
     state.page = 1;
   },
+  setListPerPage(state, val) {
+    state.taskListPerPage = state.taskList;
+  },
+  filterTaskList(state, val) {
```

Comment:
nitpicking: we might have better names than val 👀 i.e. `filter`

Reply:
Okay. Thank you